contrib/npm/archive.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Hurl (https://hurl.dev)
+ * Copyright (C) 2022 Orange
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+const { existsSync, mkdirSync } = require("fs");
```

Comment:
You an add `createWriteStream` here, use it on 57, and remove line 24.

Reply:
Thanks! I've simplified the two imports and use only the `const fs = require("fs")` (to keep the module "name spacing" when calling function). I don't know what is the best practices in Node/Javascript.